tracer/build/_build/PrepareRelease/SetAllVersions.cs

```diff
@@ -231,67 +237,67 @@ public override void Run()
 
                 // Nuke build
                 SynchronizeVersion(
-                    "build/_build/Build.cs",
+                    "tracer/build/_build/Build.cs",
```

Comment:
I think the following files, down as far as `Datadog.Trace.ClrProfiler.Native/version.h` haven't moved.

Reply:
Yes it's true but I have changed the path we provide to the `SetAllVersion` class [here](https://github.com/DataDog/dd-trace-dotnet/pull/2724/files#diff-eefdb8c0fb3de1dc1e462826cfe3f8bad3958af59a5f49182a29291920e78e5dR204) to use the root folder and not only the Tracer folder as we started to have more and more files outside the tracer folder.